src/pages/cart/CartItems.jsx

```diff
@@ -3,6 +3,8 @@ import CartCard from "../../components/CartCard";
 import PriceSummary from "../../components/PriceSummary";
```

Comment:
Ideally you should prefer named exports over default exports. 
Why?:
- It helps you keep track of what exactly you have imported
- You can rename it using 'as' and you will know that I have renamed the imported thing.
- Default exports have a loop hole you can import them using any name making it difficult to keep track.

Reply:
Thanks for mentioning, yes I do use named exports now in my current projects.
This was old code hence why I have not updated it, but good you caught it
I have updated the code now, using named exports